pennylane/operation.py

```diff
@@ -1161,51 +1161,54 @@ class Channel(Operation, abc.ABC):
     """
     # pylint: disable=abstract-method
 
-    @classmethod
+    @staticmethod
     @abc.abstractmethod
-    def _kraus_matrices(cls, *params):
+    def compute_kraus_matrices(*params, **hyperparams):  # pylint:disable=unused-argument
```

Comment:
I know we have already talked about this before, but is there any utility in moving this to `Operator`? Since I suppose anything can be represented using Kraus matrices right?

Reply:
Yes...if I had more time I'd do it :) Maybe we hold off for now!